app/lib/constants.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
     PERIOD_END: 'PeriodEnd',
     SUPPLIER: 'Supplier',
   },
-  OUTPUT_DIR: 'output',
+  OUTPUT_DIR: 'html/json',
```

Comment:
What I mean is the directory is called 'html' when it contains css, json, csv as well as html. Would it be better named 'webroot' or something along those lines?